code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -45,6 +45,11 @@
 					movement_target = null
 					stop_automated_movement = 0
 					break
+			for(var/obj/item/toy/cattoy/T in view(1,src))
+				if (!T.cooldown)
+					emote("me", 1, "bats \the [T] around with its paw!")
+					T.cooldown = 1
+					spawn(100) T.cooldown = 0

```

Comment:
@RemieRichards 

Are we no longer supposed to use world.time for cooldowns? Just a couple weeks ago I was told to use that in place of processing and spawn.